Plan/common/src/main/java/com/djrapitops/plan/extension/implementation/providers/gathering/DataValueGatherer.java

```diff
@@ -144,6 +154,10 @@ private void addValuesToBuilder(ExtensionDataBuilder dataBuilder, ExtensionMetho
             if (brokenMethods.contains(provider)) continue;
             dataBuilder.addValue(Table.class, tryToBuildTable(dataBuilder, parameters, provider));
         }
+        addValuesToBuilder2(dataBuilder, methods, parameters);
+    }
+
+    private void addValuesToBuilder2(ExtensionDataBuilder dataBuilder, ExtensionMethods methods, Parameters parameters) {
```

Comment:
This is because of "... Plan/common/src/main/java/com/djrapitops/plan/extension/implementation/providers/gathering/DataValueGatherer.java:124:5: Cyclomatic Complexity is 19 (max allowed is 17). [CyclomaticComplexity]"
 https://github.com/plan-player-analytics/Plan/actions/runs/3339786173/jobs/5529145169#step:13:255

Reply:
Alright I'll refractor it out later, feel free to add a todo comment here :)